scripts/calcRD.sh

```diff
@@ -12,11 +12,15 @@ count=$3
 
 numerator=$(echo $frd $count| awk ' { printf "%0.2f\n", ($1 * $2); } ')
 
+readsToSample=$(echo $numerator $ord| awk ' { printf "%0.2f\n", ($1 / $2); } ')
```

Comment:
Sometimes it may be easier / more readable to use python (even in-line python embedded in bash) in a WDL or script, especially if you’re dealing with floats. An example of this in GATK-SV can be found [here ](https://github.com/broadinstitute/gatk-sv/blob/0a6fcf136f632e5b324a7d8a65b9e9e7c69399ba/wdl/Module04b.wdl#L526)(not my favorite piece of code but just linking for the inline python example)